<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>My Illuminance Meter UI</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
* {
	background-color: #e0e0e5;
	font-weight: bold;
}

#MainWindow {
}

QPushButton {
	color: #404042;	
	background-color: #fff;
	font-weight: bold;
	border-radius: 7px;	
	border: 2px solid #bbb;
	padding-top: 7px;
	padding-bottom: 7px;
	padding-left: 14px;
	padding-right: 14px;
}

QPushButton:hover {	
	background-color: #eeeeef;
}

QCheckBox:indicator {
	width: 14px;
	height:14px;
}

QLabel {
	color: #404042;
}

QComboBox {
	background-color: #fefefe;	
	color: #6f707a;
	font-weight: bold;
	border-radius: 7px;	
	border: 2px solid #bbb;
	padding-top: 7px;
	padding-bottom: 7px;
	padding-left: 14px;
	padding-right: 14px;
}

QComboBox:editable {
	background-color: #fefefe;	
	margin: 5px;
	border: 2px solid #bbb;
	border-radius: 7px;	
}

QComboBox:item {
	background-color: #fefefe;
}

QComboBox:item:selected {
    padding-left: 5px;
	background-color: #e0e0e5;
}

QComboBox::drop-down {
	background-color: #fefefe;	
	subcontrol-origin: padding;
	subcontrol-position: top right;
	width: 12px;
	
	border-left: 0px solid #bbb;
	border-radius: 7px;	
	border-top-right-radius: 7px;
	border-bottom-right-radius: 7px;
	padding-right: 7px;
}

QComboBox::down-arrow {
	image: url(:/images/drop_dow_arrow_16px.png);
}

QComboBox::down-arrow:on {
	top: 1px;
	left: 1px;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1001</width>
      <height>641</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetMinAndMaxSize</enum>
     </property>
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QWidget" name="chartLuxWidget" native="true"/>
       </item>
       <item>
        <widget class="QWidget" name="chartVoltageWidget" native="true"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>7</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetNoConstraint</enum>
       </property>
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>20</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>20</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="maximumSize">
            <size>
             <width>275</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel {
	margin-left: 5px;
}</string>
           </property>
           <property name="text">
            <string>Porta USB:</string>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxUSB">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>275</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="maximumSize">
            <size>
             <width>275</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel {
	margin-top:  10px;
	margin-left: 5px;
}</string>
           </property>
           <property name="text">
            <string>Baudrate:</string>
           </property>
           <property name="margin">
            <number>2</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxBaudrate">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>275</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="maximumSize">
              <size>
               <width>275</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel {
	margin-top:  10px;
	margin-left: 5px;
}</string>
             </property>
             <property name="text">
              <string>Prescaler do Clock:</string>
             </property>
             <property name="margin">
              <number>2</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBoxPrescaler">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>275</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="maximumSize">
            <size>
             <width>275</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel {
	margin-top:  10px;
	margin-left: 5px;
}r</string>
           </property>
           <property name="text">
            <string>Porta Analógica</string>
           </property>
           <property name="margin">
            <number>2</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxPort">
           <property name="minimumSize">
            <size>
             <width>275</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>275</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonStart">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>275</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>275</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">/*
QPushButton:hover {	
	background-color: #c9ffd4;
}
*/</string>
         </property>
         <property name="text">
          <string>Começar Leitura</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonStop">
         <property name="minimumSize">
          <size>
           <width>275</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>275</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">/*
QPushButton:hover {	
	background-color: #ffc9d4;
}
*/</string>
         </property>
         <property name="text">
          <string>Parar a Leitura</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonClear">
         <property name="minimumSize">
          <size>
           <width>275</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>275</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Apagar Gráfico</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
